typescript-selenium-webdriver/README.md

```diff
@@ -2,15 +2,24 @@
 
 This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser, including how to suppress pre-existing or third-party failures using [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing).
 
+## Getting Started
+
+The individual files in the sample contain comments that explain the important parts of each file in context.
+
+Some good places to start reading are:
+
+* [tests/index.test.ts](./tests/index.test.ts): Jest test file that opens [src/index.html](./src/index.html) in a browser with Selenium and runs accessibility scans against it with axe-webdriverjs
+* [azure-pipelines.yml](./azure-pipelines.yml): Azure Pipelines config file that sets up our Continuous Integration and Pull Request builds
+* [jest.config.js](./jest.config.js): Jest configuration file that enables Typescript (using ts-jest) and test result reporting in Azure Pipelines (using jest-junit)
+
 ## Tools and libraries used
 
-* [typescript](https://www.typescriptlang.org/) to author our test code
-* [Jest](https://jestjs.io/) as our test framework, with [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing) for baselining
+* [Typescript](https://www.typescriptlang.org/) to author our test code

```

Comment:
Super-picky: TypeScript should be [Pascal cased](https://www.typescriptlang.org/)